package.json

```diff
@@ -1,80 +1,85 @@
 {
   "name": "tidepool-uploader",
-  "version": "2.22.2",
+  "version": "2.22.3-webpack-update",
   "description": "Tidepool Project Universal Uploader",
   "private": true,
-  "main": "main.js",
+  "main": "main.prod.js",
   "author": "Kent Quirk",
   "license": "BSD-2-Clause",
   "repository": "tidepool-org/uploader",
   "scripts": {
     "av-whitelist": "node ./scripts/whitelisting/av-submit.js",
     "build-docs": "./scripts/update-gh-pages.sh",
     "serve-docs": "./node_modules/.bin/gitbook serve",
-    "test": "cross-env NODE_ENV=test BABEL_DISABLE_CACHE=1 electron-mocha --renderer --retries 2 --require babel-register --recursive test/",
-    "test-debug": "cross-env NODE_ENV=test BABEL_DISABLE_CACHE=1 electron-mocha --interactive --retries 2 --require babel-register --recursive test/",
+    "test": "cross-env NODE_ENV=test BABEL_DISABLE_CACHE=1 jest",
     "test-all": "npm run lint && npm run test && npm run build",
     "lint": "eslint --cache --format=node_modules/eslint-formatter-pretty .",
     "lint-fix": "npm run lint -- --fix",
-    "hot-server": "cross-env NODE_ENV=development node --max_old_space_size=2096 -r babel-register server.js",
     "build-main": "yarn build-main-quiet --progress --profile --colors",
-    "build-main-quiet": "cross-env NODE_ENV=production node -r babel-register ./node_modules/webpack/bin/webpack --config webpack.config.electron.js",
+    "build-main-quiet": "cross-env NODE_ENV=production webpack --config webpack.config.main.prod.babel.js",
     "build-renderer": "yarn build-renderer-quiet --progress --profile --colors",
-    "build-renderer-quiet": "cross-env NODE_ENV=production node -r babel-register ./node_modules/webpack/bin/webpack --config webpack.config.production.js",
-    "build": "npm run build-main && npm run build-renderer",
-    "build-quiet": "npm run build-main-quiet && npm run build-renderer-quiet",
-    "start": "cross-env NODE_ENV=production electron ./app/",
-    "start-hot": "cross-env HOT=1 NODE_ENV=development electron -r babel-register -r babel-polyfill ./app/main.development",
+    "build-renderer-quiet": "cross-env NODE_ENV=production webpack --config webpack.config.renderer.prod.babel.js",
+    "build": "concurrently \"yarn build-main\" \"yarn build-renderer\"",
```

Comment:
Nice, this should speed up things a bit!

Reply:
As long as we have the cores available 🤞